api/v1alpha1/envoyproxy_helpers.go

```diff
@@ -37,8 +37,9 @@ func (e *EnvoyProxy) GetEnvoyProxyProvider() *EnvoyProxyProvider {
 // DefaultEnvoyProxyKubeProvider returns a new EnvoyProxyKubernetesProvider with default settings.
 func DefaultEnvoyProxyKubeProvider() *EnvoyProxyKubernetesProvider {
 	return &EnvoyProxyKubernetesProvider{
-		EnvoyDeployment: DefaultKubernetesDeployment(DefaultEnvoyProxyImage),
-		EnvoyService:    DefaultKubernetesService(),
+		EnvoyDeployment:     DefaultKubernetesDeployment(DefaultEnvoyProxyImage),
+		EnvoyService:        DefaultKubernetesService(),
+		PodDisruptionBudget: DefaultKubernetesPDB(),
```

Comment:
this would enable it by default, but looks like we'd want this to be opt in like hpa

Reply:
Indeed, i missed we want this opt in,  sure we can just remove that.